packages/mobile/src/account/FiatExchange.tsx

```diff
@@ -82,6 +78,12 @@ function FiatExchange() {
             </Text>
           </ListItem>
         )}
+        {regionCode === 'PH' && (
```

Comment:
Should we move this to a country feature flag? Or is this eventually gonna be dynamic from the merchants list?

Reply:
I can see us having different things in the `Share` screen for different countries in the future, I'll move it to a country feature flag for now and we'll see I guess?